server/services/offender/index.js

```diff
@@ -215,7 +215,7 @@ const createOffenderService = (
         throw new Error('Invalid dates supplied');
       }
 
-      if (!isBefore(startDateObj, endDateObj)) {
+      if (isBefore(endDateObj, startDateObj)) {
```

Comment:
fix this to allow creating events for today

Reply:
noticed that not on events in this service use Sentry... think it would be better to take a step back and look at better patterns for error handling than fix as this PR.